pom.xml

```diff
@@ -97,6 +98,12 @@
 			<version>${version.jackson.core}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>com.fasterxml.jackson.datatype</groupId>
+			<artifactId>jackson-datatype-jdk8</artifactId>
```

Comment:
The dependency is needed so that Jackson can properly work with the `Optional` data type.